src/test/regress/multi_mx_schedule

```diff
@@ -76,3 +76,5 @@ test: check_mx
 # in the shared memory
 # --------
 test: ensure_no_shared_connection_leak
+
+test: issue_6592
```

Comment:
The test should be higher up in the schedule. Before all these ensure_no_xxx_leak tests.